Symfony/CS/Fixer/PSR2/EofEndingFixer.php

```diff
@@ -25,13 +26,25 @@ public function fix(\SplFileInfo $file, $content)
     {
         // [Structure] A file must always end with a linefeed character
 
-        $content = rtrim($content);
+        if (empty($content)) {
+            return $content;
+        }
+
+        $tokens = Tokens::fromCode($content);
```

Comment:
if you start using tokens in this fixer, please make the whole logic manipulate tokens (as done in the master branch). It will make things easier when merging between branch